tests/integration/test_small_deployment_upgrades_edge_channel.py

```diff
@@ -25,10 +25,9 @@
 
 
 VERSION_TO_REVISION = {
+    "2.12.0": 90,
     "2.13.0": 91,
 }
-FIRST_VERSION = "2.12.0"
```

Comment:
I think we should keep this one as we're hardcoding `2.12.0` in other parts of the code.

Reply:
Yes, kept it AND use `FIRST_VERSION` as part of the main version-2-revision dict.